code/modules/events/_event_admin_setup.dm

```diff
@@ -159,3 +159,23 @@
 			chosen = FALSE
 		else
 			return ADMIN_CANCEL_EVENT
+
+/datum/event_admin_setup/multiple_choice
+	///Text shown to the admin when queried about which options they want to pick.
+	var/input_text = "Unset Text"
+	///The minimum number of choices an admin must make for this event.
+	var/min_choices = 0
```

Comment:
why isn't this `1` if `tgui_input_checkboxes`' `min_checked` argument defaults to `1`?

Reply:
So the default on tgui_input_checkbox being 1 is a holdover from it initially having a hardcoded minimum of 1. For this case the only event that uses this event_admin_setup has a minimum limit of 0 so I just thought of setting this on its parent. Its up to you I can change this var to default to 1 and add an override on the kudzu that makes it 0 if you think it'll make more sense.

Reply:
yeah that's probably fine then